pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -52,11 +52,12 @@ func newReconciler(mgr manager.Manager, status *statusmanager.StatusManager) *Re
 	configv1.Install(mgr.GetScheme())
 	operv1.Install(mgr.GetScheme())
 	return &ReconcileOperConfig{
-		client:        mgr.GetClient(),
-		scheme:        mgr.GetScheme(),
-		status:        status,
-		mapper:        mgr.GetRESTMapper(),
-		podReconciler: newPodReconciler(status),
+		client:                  mgr.GetClient(),
+		scheme:                  mgr.GetScheme(),
+		status:                  status,
+		mapper:                  mgr.GetRESTMapper(),
+		podReconciler:           newPodReconciler(status),
+		ingressConfigReconciler: newIngressConfigReconciler(mgr.GetClient()),
```

Comment:
The `podReconciler` is created here because the operconfig reconciler needs to call methods on it. The `ingressConfigReconciler` is independent and so should be created as its own top-level reconciler (added via `pkg/controller/add_networkconfig.go`

Reply:
Moved ingress config controller at the top-level.